### PR TITLE
Fixed validation in feeds integration for collect/connect sections

### PR DESCRIPTION
**Pull request by Shellyber**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5481

## Description
Fixed the Feed Integration validation for the Collect/Connect section

### CODE REVIEW

**Comment by Shellyber** [comment]
@ShahafBenYakir the `sectionorder` and the `section` field are already in the integration schema.
The Feed validation is a bit different, it is working with constants that check all the mandatory fields.
So this is the best solution for this issue.
It is that or to change the entire flow and create a new and different schema for feed integrations and change the entire validation flow accordingly

**Comment by Shellyber** [comment]
@ShahafBenYakir Test was added, please review :)

File: demisto_sdk/commands/common/tests/integration_test.py [hunk 1]
```diff
@@ -2128,6 +2128,26 @@ def test_hidden_feed_reputation_field(self):
             self.validator.all_feed_params_exist() is True
         ), "all_feed_params_exist() returns False instead True for feedReputation param"
 
+    def test_section_field_feed(self):
+        """
+        Given:
+        - Parameters of feed integration.
+        - One param with the section field
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/common/tests/integration_test.py:2135]:
```suggestion
        - Parameters of feed integration, where one parameter as a 'section: collect' property.
```

File: demisto_sdk/commands/common/tests/integration_test.py [hunk 2]
```diff
@@ -2128,6 +2128,26 @@ def test_hidden_feed_reputation_field(self):
...
+
+        When:
+        - Integration has all feed required params, and validation won't fail on the section field.
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/common/tests/integration_test.py:2138]:
```suggestion
        - Integration has all feed required params and running all_feed_params_exist on it.
```

File: demisto_sdk/commands/common/tests/integration_test.py [hunk 3]
```diff
@@ -2128,6 +2128,26 @@ def test_hidden_feed_reputation_field(self):
...
+
+        Then:
+        - Ensure that all_feed_params_exists() returns true.
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/common/tests/integration_test.py:2141]:
```suggestion
        - Ensure that all_feed_params_exists() returns true, which means validation did not fail on the additional section field.
```

**Review by ShahafBenYakir — APPROVED** [review]
Great

**Pull request merged**